### PR TITLE
fixed the product page URL issue

### PR DESCRIPTION
**Pull request by WebkulOpencart**
fix [https://github.com/opencart/opencart/issues/9363](https://github.com/opencart/opencart/issues/9363)

### CODE REVIEW

File: upload/catalog/model/catalog/product.php [hunk 1]
```diff
@@ -536,4 +536,13 @@ public function getTotalProductSpecials() {
...
 		}
 	}
+
+	public function checkProductCategory($product_id, $category_ids) {
+
+		$category_id = implode("', '", $category_ids);
+
+		$query = $this->db->query("SELECT * FROM " . DB_PREFIX . "product_to_category WHERE product_id = '" . (int)$product_id . "' AND category_id IN('$category_id')");
+
+		return $query->row;
```
Comment (TheCartpenter) [inline upload/catalog/model/catalog/product.php:546]: ```return $query->rows;```

As for the IN statement, I would suggest the use of catalog/model/account/download.php file the way the $implode with AND, OR statement are being used originally.

Comment (WebkulOpencart) [reply upload/catalog/model/catalog/product.php:546]: I am checking for the same.

Comment (TheCartpenter) [reply upload/catalog/model/catalog/product.php:546]: Not sure what that means but it seem you have a few 'for the same' in your replies there. Does this mean you'll make the same or you believe it's the same?

File: upload/catalog/model/catalog/product.php [hunk 2]
```diff
@@ -536,4 +536,13 @@ public function getTotalProductSpecials() {
 			return 0;
 		}
 	}
+
+	public function checkProductCategory($product_id, $category_ids) {
+
```
Comment (TheCartpenter) [inline upload/catalog/model/catalog/product.php:541]: Remove empty space. No models have spaced areas between the public function and the next line.

**Pull request merged**